packages/ComponentLibrary/src/components/FormView/index.tsx

```diff
@@ -0,0 +1,283 @@
+import { useCallback, useMemo, useState } from 'react';
+import {
+  TextField,
+  Grid,
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  Typography,
+  Box,
+  FormControl,
+  FormControlLabel,
+  Checkbox,
+  styled,
+} from '@mui/material';
+import ChevronDown from '../../assets/icons/chevron-down.svg';
+import { SearchOutlined } from '@mui/icons-material';
+import { Select, TextInputBase, theme } from '..';
+import { topFilms } from '../../../../storybook/src/stories/Components/Input/Select/mock';
+import {
+  Organization,
+  FieldDefinition,
+  Section,
+  BaseFieldDefinition,
+} from '../../../../storybook/src/stories/Components/Table/types';
+import { FormViewProps } from './types';
+import { defaultFill, styles, sx } from './styles';
+import IconButton from '../IconButton';
+import InfoIcon from '@mui/icons-material/Info';
+import PrimaryTabs from '../PrimaryTab';
+import { TabItem } from '../PrimaryTab/types';
+
+const defaultIcon = (
+  <ChevronDown fill={theme.palette.baselineColor.neutral[80]} />
+);
+
+const FieldLabel: React.FC<{ label: string; required?: boolean }> = ({
```

Comment:
Can't the 3 components be separated in another file? It would be much easier to read.